pom.xml

```diff
@@ -48,7 +48,7 @@
 		<profile>
 			<id>eclipselink-next</id>
 			<properties>
-				<eclipselink>2.6.3-SNAPSHOT</eclipselink>
+				<eclipselink>2.6.5-SNAPSHOT</eclipselink>
```

Comment:
Looks like we don't actually pull up the version of the version used for standard compilation to 2.6.4 then, do we? If we upgrade Eclipselink dependencies, let's upgrade them entirely, to the latest version available.